scripts/Engineering/gui/engineering_diffraction/tabs/fitting/data_handling/data_view.py

```diff
@@ -53,6 +84,9 @@ def get_filenames_valid(self):
     def get_add_to_plot(self):
         return self.check_addToPlot.isChecked()
 
+    def get_selected_rows(self):
+        return list(set(index.row() for index in self.table_selection.selectedIndexes()))
```

Comment:
```suggestion
        return [index.row() for index in self.table_selection.selectedIndexes()]
```

In the Qt docs for `selectedIndexes()`:
> This convenience function returns a list of all selected and non-hidden item indexes in the view. The list contains **no duplicates**, and is not sorted.